apps/emqx_gateway/src/emqx_gateway_api_listeners.erl

```diff
@@ -553,14 +588,50 @@ params_paging_in_qs() ->
 roots() ->
     [listener].
 
+fields(listener_node_status) ->
+    [
+        {current_connections, mk(non_neg_integer(), #{desc => ?DESC(current_connections)})},
```

Comment:
miss `max_connections` field

Reply:
The 'max_connections'  already exist in emqx_gateway_api:fields(xxx_listeners)